### PR TITLE
Adding quick intro and install section in home page

### PR DESCRIPTION
**Pull request by sandeep-krishnamurthy**
* Added quick intro - What is MXNet? and Quick Install steps in MXNet home page.
* Tested on local machine for content and CSS (minimize, maximize, scroll, text size etc..)

Credits: Quick intro content and wireframe is contributed by @zackchase 

Snapshot is attached
screen shot 2017-04-05 at 3 46 31 pm

### CODE REVIEW

**Comment by sandeep-krishnamurthy** [comment]
@madjam @zackchase @piiswrong - Can you please review this change?

**Comment by piiswrong** [comment]
more details -> Other OS & build from source

**Comment by sandeep-krishnamurthy** [comment]
@piiswrong - Makes sense. Changed from more details.. to "Other OS and building from source.."

**Comment by zackchase** [comment]
Perhaps it's premature for a pull request on this since the Why MXNet doc
is not part of the main repo yet (still on the doc staging repo
https://github.com/zackchase/mxnet-docs)

**Comment by mli** [comment]
mac osx -> macOS

ubuntu, macos, and docker are quite different options, it doesn't make sense to me to put them side by side. also the first two options only install the python package. i don't think we are ready to to put them in the front page if pre-build for other languages are not ready now

File: docs/_static/mxnet-theme/index.html [hunk 1]
```diff
@@ -21,6 +24,54 @@
...
 </div>
 
+<!-- Start - Intro and Quick install section -->
+<div class="section-intro">
+  <div class="container">
+    <div class="row">
+      <div class="section-intro-col">
+        <h2> What is MXNet? </h2>
+        <p align="justify">
+          Deep learning denotes the modern incarnation of neural networks, and it's the technology behind recent breakthroughs in self-driving cars, machine translation, speech recognition and more. While widespread interest in deep learning took off in 2012, deep learning has already become an indispensable tool for countless industries. MXNet is an open-source deep learning framework that allows you to rapidly define, train, and deploy deep neural networks on a wide array of devices, including from GPU-powered cloud infrastructure to mobile devices. Additionally, MXNet supports a flexible programming model and multiple languages.
```
Comment (madjam) [inline docs/_static/mxnet-theme/index.html]: ", including from GPU-powered" 

including feels redundant.

**Comment by zackchase** [comment]
Yes this is a typo. The paragraph needs some light revisions and we also
need to be serious about better styling before merging

**Comment by mli** [comment]
as per our discussion we are not going to merge this pr at this moment. so i'm going to close it now. feel free to reopen it later when we are ready to update the front page

**Pull request closed without merging**